Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -67,27 +63,27 @@ public struct MessageView: View {
         }
     }
     
+    public enum Defaults {
+        static let hideMessagesWithRoles: Set<Chat.Role> = [.system, .function]
+    }
     
-    /// - Parameters:
-    ///   - chat: The chat message that should be displayed.
-    ///   - hideSystemMessages: If system messages should be hidden from the chat overview.
-    public init(_ chat: Chat, hideSystemMessages: Bool = true) {
+    public init(_ chat: Chat, hideMessagesWithRoles: Set<Chat.Role>) {
         self.chat = chat
-        self.hideSystemMessages = hideSystemMessages
+        self.hideMessagesWithRoles = MessageView.Defaults.hideMessagesWithRoles
     }
 }
 
-
 struct MessageView_Previews: PreviewProvider {
     static var previews: some View {
         ScrollView {
             VStack {
-                MessageView(Chat(role: .system, content: "System Message!"), hideSystemMessages: false)
-                MessageView(Chat(role: .system, content: "System Message (hidden)!"))
-                MessageView(Chat(role: .user, content: "User Message!"))
-                MessageView(Chat(role: .assistant, content: "Assistant Message!"))
+                MessageView(Chat(role: .system, content: "System Message!"), hideMessagesWithRoles: MessageView.Defaults.hideMessagesWithRoles)
```

Comment:
I can not see that the current functionality is working if you look at the previews. If you want this to be shown in the previews, you will have to e.g. use something like this and the system message will not be hidden:
```suggestion
                MessageView(Chat(role: .system, content: "System Message!"), hideMessagesWithRoles: [.function])
```